rsync2/PKGBUILD

```diff
@@ -0,0 +1,63 @@
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=rsync2
+pkgver=3.1.3dev_msys2.7.0_r2
+pkgrel=0
+pkgdesc="A file transfer program to keep remote files in sync"
+arch=('i686' 'x86_64')
+url="https://github.com/jbenden/rsync"
+groups=('net-utils')
+license=('GPL3')
+depends=('libiconv' 'msys2-w32api-runtime')
+makedepends=('msys2-w32api-headers' 'perl' 'yodl')
+conflicts=('rsync')
+provides=('rsync')
+pkgsname=rsync
+pkgsver="3.1.3dev-msys2.7.0-r2"
+source=("https://codeload.github.com/jbenden/rsync/tar.gz/v3.1.3dev-msys2.7.0-r2"
```

Comment:
The project is from a development Git repository, because it is from the upstream Samba's Git repository. Put another way, there is not a release version to patch against.  I have been trying to get Samba to incorporate the patches in to their sources; but they have not responded to the [ticket](https://bugzilla.samba.org/show_bug.cgi?id=12755) I've posted a while ago.